app/src/views/Home.vue

```diff
@@ -20,9 +36,18 @@
       <v-form class="mt-12 mb-12">
         <v-row justify="center">
           <v-col cols="12" md="8">
-            <search-bar
```

Comment:
Why did you revert the refactoring into a component?

Reply:
Unlucky timing. I wasnt merging these changes onto my local branch. Gonna revert that